script.js

```diff
@@ -1,12 +1,106 @@
-let colorlist = ['gold', 'yellow', 'turquoise', 'red']
+let myImages = ['Winnie-the-Pooh.png', 'tinker-bell.png', 'simba.png', 'dumbo.png', 'Max-andRuby.png', 'disney-princesses.png']
```

Comment:
I think you have small typo here: `'Max-andRuby.png'` should be `'Max-and-Ruby.png'`